src/base/oven.lua

```diff
@@ -43,7 +43,7 @@
 ---
 
 	function oven.bake()
-		p.container.bakeChildren(p.api.rootContainer())
+		p.container.bake(p.api.rootContainer())
```

Comment:
I think this is what broke your build, the function `bake` is in the `p.global` namespace.